example_project/templates/auth/login-error.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
should we use the gov styling?

Reply:
This is just a stand-in template so the settings will load. Each app will provide the template to use